src/main/java/Task.java

```diff
@@ -0,0 +1,43 @@
+public class Task {
```

Comment:
consider renaming this to todo or create new subclass todo.
task and todo are used interchangebly which can be confusing